src/server/auth/db.go

```diff
@@ -0,0 +1,20 @@
+package auth
+
+import (
+	"github.com/jmoiron/sqlx"
+	"golang.org/x/net/context"
+)
+
+// TODO(acohen4): Decide on hash as primary key or unique column
+// CreateTokensTable sets up the postgres table which tracks active clusters
+func CreateAuthTokensTable(ctx context.Context, tx *sqlx.Tx) error {
+	_, err := tx.ExecContext(ctx, `
+CREATE TABLE IF NOT EXISTS auth.auth_tokens (
+	token_hash VARCHAR(4096) PRIMARY KEY,
```

Comment:
@actgardner any thoughts on having token_hash be a `primary key` vs a `unique field`?

Reply:
Primary key makes sense to me - the postgres docs say it's just uniqueness + not null, both of which apply to the token hash.